resources/default_pipeline_environment.yml

```diff
@@ -156,6 +156,9 @@ steps:
         fail:
           high: '0'
     archive: false
+  cloudFoundryDeleteService:
```

Comment:
Added defaults to `cloudFoundryDeleteService .yaml` in the container section. However, I cannot delete config from `default_pipeline_environment.yml` without running into error. Hence, marking this as resolved since config has also been added to cfDeleteService.yaml file.